projects/addon-editor/tokens/editor-extensions.ts

```diff
@@ -0,0 +1,6 @@
+import {InjectionToken} from '@angular/core';
+import {Extension, Mark, Node} from '@tiptap/core';
+
+export const TUI_EDITOR_EXTENSIONS = new InjectionToken<
+    ReadonlyArray<Extension | Mark | Node>
```

Comment:
What are those types?

Reply:
These are editor entities that can be used as an extension